apache-jena-osgi/jena-osgi/src/main/java/org/apache/jena/osgi/Activator.java

```diff
@@ -0,0 +1,47 @@
+package org.apache.jena.osgi;
+
+import org.apache.jena.system.JenaSubsystemRegistry;
+import org.apache.jena.system.JenaSubsystemRegistryBasic;
+import org.apache.jena.system.JenaSystem;
+import org.osgi.framework.BundleActivator;
+import org.osgi.framework.BundleContext;
+
+public class Activator implements BundleActivator {
+	
+	/* 
+	 * Based on http://svn.apache.org/repos/asf/aries/trunk/spi-fly/spi-fly-examples/spi-fly-example-provider-consumer-bundle/src/main/java/org/apache/aries/spifly/pc/bundle/Activator.java
+	 * the Activator#start() waits for bundle extension by Aries SPI Fly, configures JenaSystem logging and requests for initialization.     
+	 */
+	public void start(BundleContext context) throws Exception {
+
+		Thread t = new Thread(new Runnable() {
+			@Override
+			public void run() {
+				try {
+					Thread.sleep(500);
+				} catch (InterruptedException e) {
+				}				
+				setUpJena();
+			}
+		});
+		t.start();
+	}
+
+	private void setUpJena() {
+		JenaSubsystemRegistry r = new JenaSubsystemRegistryBasic() {
+			@Override
+			public void load() {
+				super.load();
+			}
+		};
```

Comment:
The OSGi users are used to this process of a painfull dependcy resolution, luckily this is due only once at the beginning. While the package dependencies are obvoius from the manifest, we should support the resolution of the correct (OSGi-aware) Maven archives. This could be done by specifying a provisioning configuration, e.g. an [Apache Karaf Fature](https://karaf.apache.org/manual/latest/users-guide/provisioning.html) or [Equinox/Tycho](https://wiki.eclipse.org/Tycho/Reference_Card) feature. I'll need some time to prepare them and to set-up integration tests for these platforms (but this is a prerequisite to prove the portability).